benchmark/fabfile.py

```diff
@@ -21,7 +21,8 @@ def local(ctx, debug=True):
         'workers': 1,
         'rate': 50_000,
         'tx_size': 512,
-        'duration': 20,
+        'duration': 300,
```

Comment:
While I 100% understand the longer run time for memory profiling, I'd love to not change the default run time for non-memory-profiling benches.